client/my-sites/customer-home/main.jsx

```diff
@@ -300,6 +303,7 @@ const connectHome = connect(
 			isEstablishedSite: moment().isAfter( moment( createdAt ).add( 2, 'days' ) ),
 			siteIsUnlaunched: isUnlaunchedSite( state, siteId ),
 			user,
+			cards: getHomeLayout( state, siteId ),
```

Comment:
@kwight we could maybe rename this to layout too, but not a big deal right now since it's unused 🙂